cmd/monitor_doers.go

```diff
@@ -0,0 +1,128 @@
+// Copyright © 2019 The VirusTotal CLI authors. All Rights Reserved.
```

Comment:
I don't think we need a separate file for this, it's ok to put this in `monitor.go`.